lib/execution_engine2/SDKMethodRunner.py

```diff
@@ -835,9 +835,14 @@ def check_jobs(self, job_ids, check_permission=True, projection=None):
 
             job_states[str(job.id)] = mongo_rec
 
+        job_states = dict(sorted(job_states.items()))
+
+        if return_list:
+            job_states = list(job_states.values())
```

Comment:
What would happen if the string "False" was passed in here?

Reply:
👍 hopefully, no one does that. I found your `parse_bool_from_string`. 💯

Reply:
Ah did you use it? I swear I saw you use it in a diff commit and now its gone from this PR or maybe its just too late at night right now.

Reply:
@bio-boris hmmm no it's there. `if return_list is not None and SDKMethodRunner.parse_bool_from_string(return_list):`